src/components/Button/Button.jsx

```diff
@@ -114,7 +114,7 @@ const Button = createClass({
 
 		return (
 			<button
-				{...passThroughs}
+				{...omitProps(passThroughs, Button, ['callbackId'])}
```

Comment:
here omitProps is just a call so the whole cost of creating partial function scenario most likely doesn't apply here...
as for memoization - a reselect.createSelector wrapped function may optimize - but probably not worth it at a lower level...